dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/actionlet/PushNowActionletTest.java

```diff
@@ -0,0 +1,215 @@
+import com.dotmarketing.portlets.workflows.business.BaseWorkflowIntegrationTest;
+import com.dotmarketing.portlets.workflows.business.WorkflowAPI;
+import com.dotmarketing.portlets.workflows.model.MultiKeyValue;
+import com.dotmarketing.portlets.workflows.model.MultiSelectionWorkflowActionletParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionClass;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionletParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowProcessor;
+import com.liferay.portal.model.User;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.validation.constraints.AssertTrue;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PushNowActionletTest extends BaseWorkflowIntegrationTest {
+
+    private static CreateSchemeStepActionResult schemeStepActionResult = null;
+    private static WorkflowAPI workflowAPI = null;
+    private static ContentletAPI contentletAPI = null;
+    private static ContentTypeAPI contentTypeAPI = null;
+    private static ContentType type = null;
+    private static User adminUser;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+
+        IntegrationTestInitService.getInstance().init();
+        workflowAPI = APILocator.getWorkflowAPI();
+        contentTypeAPI = APILocator.getContentTypeAPI(APILocator.systemUser());
+        contentletAPI = APILocator.getContentletAPI();
+        final long currentTime = System.currentTimeMillis();
+        adminUser = TestUserUtils.getUser(TestUserUtils.getOrCreateAdminRole(),"testPushNow@test.com"+currentTime,"testPushNowActionlet"+currentTime,"testPushNowActionlet"+currentTime,"testPushNowActionlet"+currentTime);
+
+    }
+
+    private void createWorkflowWithPushNowActionlet(final String environmentNameParam, final String filterKeyParam)
+            throws DotSecurityException, AlreadyExistException, DotDataException, InstantiationException, IllegalAccessException {
+        // Create the scheme and actions. This method allows you to add just one sub-action
+        final long sysTime = System.currentTimeMillis();
+        schemeStepActionResult = createSchemeStepActionActionlet
+                ("itPushNowScheme_" + sysTime, "step1", "action1",
+                        CheckinContentActionlet.class);
+        // Add the Push Now sub-action for this test
+        addActionletToAction(schemeStepActionResult.getAction().getId(),
+                PushNowActionlet.class, 1);
+        // Add the required parameters of the sub-action
+        final List<WorkflowActionClass> actionletClasses = getActionletsFromAction(
+                schemeStepActionResult.getAction());
+        WorkflowActionClass workflowActionClass = actionletClasses.get(1);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'workflowActionClass' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5458951)